### PR TITLE
Add localization (English <-> French)

### PR DESCRIPTION
**Pull request by acchiang**
https://user-images.githubusercontent.com/47487758/129156923-f7370b71-b8e9-43f0-8cbc-d298ce6ae763.mov

Works across all pages.

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
-import React from "react";
+import React, {Suspense} from "react";
 import ReactDOM from "react-dom";
 import App from "pages/App";
 import reportWebVitals from "./utils/reportWebVitals";
 
 import "./index.css";
 
+// import i18n (needs to be bundled ;))
```
Comment (rzyxie) [inline client/src/index.js:8]: remove comment?

**Comment by rzyxie** [comment]
things still in english: 
- /register page (in Login.js) 
- alerts for copy URL and change event name 

food for thought:
can we move the language toggle into the main create session and login page instead of charcuterie? makes more sense from the user's perspective, unless this was by design

File: client/src/index.js (hunk 1 above)
Comment (acchiang) [reply client/src/index.js:8]: that came from the docs but ya oke i do it

**Comment by acchiang** [comment]
thanks @rzyxie i do that in a sec and let u know when it ready

**Comment by acchiang** [comment]
https://user-images.githubusercontent.com/47487758/129168677-f5d87226-8e91-4a9e-9954-4428da2f7dc4.mov

**Review by rzyxie — APPROVED** [review]

**Pull request merged**